modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/view/DeployedService.java

```diff
@@ -125,4 +127,7 @@ public class DeployedService {
 
     @Schema(description = "Details of the service configuration.")
     private ServiceConfigurationDetails serviceConfigurationDetails;
+
+    @Schema(description = "manage service action.")
+    private List<ServiceAction> serviceActions;
```

Comment:
these are the service actions from the template. not the service actions ordered by the end user. 
If the end user want to display the service actions that he can order, then we must call the getOrderableServiceDetailsByServiceId which must return the service actions.